.gitlab-ci.yml

```diff
@@ -25,18 +26,25 @@ default:
         CUSTOM_CI_BUILDS_DIR: "/usr/WS1/$$USER/gitlab-runner-builds-dir"
         # Note: the above will not work with /usr/workspace, you must specify WS1 or WS2
 
+## Job Specifications
 .test-core:
     extends: .lc-variables
     variables:
         PYTHON: "/usr/bin/python3"
         debug: t
         FLUX_TESTS_LOGFILE: t
     script:
+        - pwd
+        - echo $CI_DIRECTORY
+        - cd ${CORE_BUILD_DIR}
+        - pwd
```

Comment:
Is this debugging that wasn't meant to be committed?

Reply:
Whoops, thanks.